src/docker-compose.yml

```diff
@@ -25,6 +25,7 @@ services:
     container_name: products
     environment:
       - AWS_REGION
+      - AWS_DEFAULT_REGION
```

Comment:
The products service is written in golang and the [AWS go SDK looks for AWS_REGION](https://github.com/aws/aws-sdk-go/issues/2103). Is AWS_DEFAULT_REGION needed for the recommendations service and not products service?